src/Crypt32.Shared/Crypt32+CRYPT_KEY_PROV_INFO.cs

```diff
@@ -1,19 +1,52 @@
-﻿namespace PInvoke
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
 {
     using System;
     using System.Runtime.InteropServices;
 
     public partial class Crypt32
     {
+        /// <summary>
+        /// The CRYPT_KEY_PROV_INFO structure contains information about a key container within a cryptographic service provider (CSP).
+        /// </summary>
         [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
-        public struct CRYPT_KEY_PROV_INFO
+        public unsafe struct CRYPT_KEY_PROV_INFO
```

Comment:
Can you add `[OfferIntPtrPropertyAccessors]` to this struct and set the Custom Tool property for the file? This was previously missing from our contributing.md file (sorry!) but I've corrected it so you can read about it in the last couple paragraphs of [this section](https://github.com/AArnott/pinvoke/blob/master/CONTRIBUTING.md#struct-field-types).

This will make your struct consumable by VB.NET programmers.